sqlglot/parser.py

```diff
@@ -2969,6 +2969,10 @@ def _parse_select(
             if into:
                 this.set("into", into)
 
+            bulk_collect_into = self._parse_bulk_collect_into()
+            if bulk_collect_into:
+                this.set("bulk_collect_into", bulk_collect_into)
```

Comment:
you need to to make bulk_collect_into a first class property of select

Reply:
can you have both into and bulk_into? if not, then i'd just use the into statement, perhaps you could just overide the into parser of oracle

Reply:
@tobymao No, you can only have INTO or BULK COLLECT INTO, not both.

However, from an AST diff perspective, I think it would be useful to be able to make a distinction between these two, as they are semantically different.